### PR TITLE
Adds some more literal flavor text to some toxins and drugs.

### PR DESCRIPTION
**Pull request by itseasytosee**
## About The Pull Request
I added some more tastes for some toxins so everything doesn't just taste the default "bitter" and a few for drugs. I also added a couple on mix messages such as  "A silent scream is heard in the distance" for the mixing of mime's bane

## Why It's Good For The Game
It gives you a second of wondering why the chaplains kool-aid tastes like almonds before you die.

## Changelog
:cl: itseasytosee
add: adds some unique taste for some common toxins and on mix messages
/:cl:

### CODE REVIEW

File: code/modules/reagents/chemistry/recipes/toxins.dm [hunk 1]
```diff
@@ -75,6 +77,7 @@
 	id = /datum/reagent/toxin/mutetoxin
 	results = list(/datum/reagent/toxin/mutetoxin = 2)
 	required_reagents = list(/datum/reagent/uranium = 2, /datum/reagent/water = 1, /datum/reagent/carbon = 1)
+	mix_message = "<span class='danger'>The soulution releases a vapor tht makes your tounge and vocal cords tense up.</span>"
```
Comment (Buggy123) [inline code/modules/reagents/chemistry/recipes/toxins.dm]: "tht"

Comment (itseasytosee) [reply code/modules/reagents/chemistry/recipes/toxins.dm]: thanks

Comment (Qustinnus) [inline code/modules/reagents/chemistry/recipes/toxins.dm]:
```suggestion
	mix_message = "<span class='danger'>The solution releases a vapor that makes your tongue and vocal cords tense up.</span>"
```

File: code/modules/reagents/chemistry/recipes/toxins.dm [hunk 2]
```diff
@@ -39,6 +40,7 @@
 	results = list(/datum/reagent/toxin/acid/nitracid = 2)
 	required_reagents = list(/datum/reagent/toxin/acid/fluacid = 1, /datum/reagent/nitrogen = 1, /datum/reagent/oxygen = 1)
 	required_temp = 380
+	mix_message = "<span class='danger'>A strong acrid smell fills the air.</span>"
```
Comment (Qustinnus) [inline code/modules/reagents/chemistry/recipes/toxins.dm]:
```suggestion
	mix_message = "<span class='danger'>A strong acid smell fills the air.</span>"
```

File: code/modules/reagents/chemistry/reagents/drug_reagents.dm [hunk 3]
```diff
@@ -13,7 +13,7 @@
 	description = "An illegal chemical compound used as drug."
 	color = "#60A584" // rgb: 96, 165, 132
 	overdose_threshold = 30
-
+	taste_description = "your own tounge"
```
Comment (Couls) [inline code/modules/reagents/chemistry/reagents/drug_reagents.dm]:
```suggestion
	taste_description = "your own tongue"
```

File: code/modules/reagents/chemistry/recipes/toxins.dm (hunk 2 above)
Comment (RaveRadbury) [reply code/modules/reagents/chemistry/recipes/toxins.dm]: https://www.merriam-webster.com/dictionary/acrid

File: code/modules/reagents/chemistry/reagents/toxin_reagents.dm [hunk 4]
```diff
@@ -231,6 +231,7 @@
 	color = "#49002E" // rgb: 73, 0, 46
 	toxpwr = 1
 	taste_mult = 1
+	taste_description = "hot sause"
```
Comment (LemonInTheDark) [inline code/modules/reagents/chemistry/reagents/toxin_reagents.dm]:
```suggestion
	taste_description = "hot sauce"
```

File: code/modules/reagents/chemistry/recipes/toxins.dm [hunk 5]
```diff
@@ -12,14 +12,15 @@
...
+	mix_message = "<span class='danger'>The space drugs burn away, leaving behind a thick residue.</span>"
 
 /datum/chemical_reaction/cyanide
 	name = "Cyanide"
 	id = /datum/reagent/toxin/cyanide
 	results = list(/datum/reagent/toxin/cyanide = 3)
 	required_reagents = list(/datum/reagent/fuel/oil = 1, /datum/reagent/ammonia = 1, /datum/reagent/oxygen = 1)
 	required_temp = 380
-
+a
```
Comment (LemonInTheDark) [inline code/modules/reagents/chemistry/recipes/toxins.dm]: wat

Comment (Qustinnus) [reply code/modules/reagents/chemistry/recipes/toxins.dm]: b

File: code/modules/reagents/chemistry/recipes/toxins.dm (hunk 2 above)
Comment (Qustinnus) [reply code/modules/reagents/chemistry/recipes/toxins.dm]: huh, til

File: code/modules/reagents/chemistry/recipes/toxins.dm (hunk 5 above)
Comment (Fikou) [reply code/modules/reagents/chemistry/recipes/toxins.dm]: h

**Comment by itseasytosee** [comment]
I impress my self with some of the errors I make.

File: code/modules/reagents/chemistry/recipes/toxins.dm [hunk 6]
```diff
@@ -75,6 +77,7 @@
 	id = /datum/reagent/toxin/mutetoxin
 	results = list(/datum/reagent/toxin/mutetoxin = 2)
 	required_reagents = list(/datum/reagent/uranium = 2, /datum/reagent/water = 1, /datum/reagent/carbon = 1)
+	mix_message = "<span class='danger'>The soulution releases a vapor that makes your tongue and vocal cords tense up.</span>"
```
Comment (spessbandit) [inline code/modules/reagents/chemistry/recipes/toxins.dm]: solution

File: code/modules/reagents/chemistry/recipes/toxins.dm [hunk 7]
```diff
@@ -119,6 +122,7 @@
 	id = /datum/reagent/toxin/mimesbane
 	results = list(/datum/reagent/toxin/mimesbane = 3)
 	required_reagents = list(/datum/reagent/uranium/radium = 1, /datum/reagent/toxin/mutetoxin = 1, /datum/reagent/consumable/nothing = 1)
+	mix_message = "<span class='danger'>A silent scream is heard in the distance.</span>"
```
Comment (spessbandit) [inline code/modules/reagents/chemistry/recipes/toxins.dm]: >silent
>heard

**Comment by ExcessiveUseOfCobblestone** [comment]
Use suggestions on grammar reviews

**Comment by Krysonism** [comment]
The reason i don't like this is because it makes poisoning food and drinks harder. It is already really hard to do effectively.

Many chemicals in real life have an indistinct bitter taste, meme taste descriptions should be reserved for the most powerful poisons.

**Comment by itseasytosee** [comment]
All of the chemical that I have changed the taste of actually have the taste that I have described and I have avoid quite a few that have the bitter taste which is the default. As for the poisoning thing that is a fair criticism. In fact I might restrict the flavor to the drugs only but I need to think.

**Pull request closed without merging**